src/table/TableHeadWrapper.jsx

```diff
@@ -43,17 +43,17 @@ function TableHeadWrapper({
 }) {
   const headStyle = useMemo(() => getHeadStyle(layout, theme), [layout, theme.name()]);
   const classes = useStyles(headStyle);
-  const SORT_NOTATIONS = useMemo(() => ({
-    asc: translator.get('SNTable.SortLabel.SortedAscending'),
-    desc: translator.get('SNTable.SortLabel.SortedDescending'),
-  }));
+  // const SORT_NOTATIONS = useMemo(() => ({
+  //   asc: translator.get('SNTable.SortLabel.SortedAscending'),
+  //   desc: translator.get('SNTable.SortLabel.SortedDescending'),
+  // }));
```

Comment:
this part is language specific thing, and I'm thinking about if we rely on aria-sort  to handle table sort and considering it announces sort direction ( sorted ascending/descending ) which is actually i’m having now in my local branch, we will not going to need translations for SNTable.SortLabel.SortedAscending and  SNTable.SortLabel.SortedDescending  respectively. that is why i'm commenting them here in my draft PR.
in that case, will browser/users OS announce the sort directions in the users selected choice of language?